src/controllers/userTransform.ts

```diff
@@ -37,10 +37,20 @@ export class UserTransformController {
     );
     const requestSize = Number(ctx.request.get('content-length'));
     const events = ctx.request.body as ProcessorTransformationRequest[];
-    const processedRespone: UserTransformationServiceResponse =
-      await UserTransformService.transformRoutine(events, ctx.state.features, requestSize);
-    ctx.body = processedRespone.transformedEvents;
-    ControllerUtility.postProcess(ctx, processedRespone.retryStatus);
+
+    let processedResponse: UserTransformationServiceResponse;
+    if (process.env.USE_PISCINA === 'true') {
```

Comment:
[question] @abhimanyubabbar @satishrudderstack does it make sense to just randomly access environment variables throughout the code? Maybe this belongs in the context? I'm not familiar with this codebase but if there is already a pattern in place for this it might make sense to follow it unless it makes things too difficult.